packages/shared/src/components/cards/PostCardHeader.tsx

```diff
@@ -33,31 +40,45 @@ export const PostCardHeader = ({
   source,
   postLink,
   openNewTab,
+  showFeedback,
 }: CardHeaderProps): ReactElement => {
   const isFeedPreview = useFeedPreviewMode();
+  const { highlightBookmarkedPost } = useBookmarkProvider({
+    bookmarked: post.bookmarked && !showFeedback,
```

Comment:
What is the showFeedback based on?

Reply:
Is about the feedback loop, so if user reads any article that was bookmarked and feeback loop shows, these colors to now show as they look weird. Once feedback loop is removed these get back there if the post is still there